tsl/src/compression/compression.c

```diff
@@ -1466,7 +1466,7 @@ Datum
 tsl_compressed_data_recv(PG_FUNCTION_ARGS)
 {
 	StringInfo buf = (StringInfo) PG_GETARG_POINTER(0);
-	CompressedDataHeader header = { { 0 } };
+	CompressedDataHeader header = { .vl_len_ = { 0 } };
```

Comment:
clang-15 doesn't like double nested braces, and gcc 4.8.5 doesn't like single braces, so this is how we have to compromise to avoid warnings

Also see this PR https://github.com/timescale/timescaledb/pull/4867/files#r1004751330